packages/block/package.json

```diff
@@ -51,6 +51,7 @@
     "ethereum-cryptography": "^3.1.0"
   },
   "devDependencies": {
+    "@ethereumjs/testdata": "1.0.0",
```

Comment:
Thought we wanted to call this just `testdata` for simplicity? Or has this drawbacks?